coredb-operator/tests/integration_tests.rs

```diff
@@ -0,0 +1,103 @@
+// Include the #[ignore] macro on slow tests.
+// That way, 'cargo test' does not run them by default.
+// To run just these tests, use 'cargo test -- --ignored'
+// To run all tests, use 'cargo test -- --include-ignored'
+//
+// https://doc.rust-lang.org/book/ch11-02-running-tests.html
+//
+// These tests assume there is already kubernetes running and you have a context configured.
+// It also assumes that the CRD(s) and operator are already installed for this cluster.
+// In this way, it can be used as a conformance test on a target, separate from installation.
+
+#[cfg(test)]
+mod test {
+
+    use controller::CoreDB;
+    use k8s_openapi::api::core::v1::{Namespace, Pod};
+    use kube::{
+        api::{Patch, PatchParams},
+        runtime::wait::{await_condition, conditions},
+        Api, Client, Config,
+    };
+    use rand::Rng;
+
+    #[tokio::test]
+    #[ignore]
+    async fn functional_test_basic_create() {
```

Comment:
this test deploys a CoreDB into the cluster we are connected to, then checks that a pod starts